README.md

```diff
@@ -173,13 +173,17 @@ High level overview of upcoming Operation Code goals.  This is the source of upc
 
 We use [Apiary](https://apiary.io/) for our API documentation, API mocking server, etc.
 
+The API blueprint file is at `apiary.apib`, and the documentation itself is at http://docs.operationcodeapi.apiary.io/
```

Comment:
I'd suggest chaining the first part of this sentence to:
"The API blueprint file is located at operationcode_backend/apiary.apib...".  When I first read this I thought apiary.apib was something I was supposed to be searching for on the Apiary website under Operation Code.